install/kubernetes/istio-auth.yaml

```diff
@@ -1173,6 +1173,22 @@ spec:
         ports:
         - containerPort: 80
         - containerPort: 443
+        readinessProbe:
+          exec:
+            command:
+            - curl http://localhost:15000/clusters && curl http://localhost:15000/listeners && curl http://localhost:15000/routes
```

Comment:
1 url should be enough for a liveness check no? (Does envoy have a /ready or one of those is the last one ready)

Reply:
you are right, one is enough, will change.
It does not have /ready, we can suggest an improvement.